packages/formatjs/src/lib.rs

```diff
@@ -23,11 +24,14 @@ pub fn process(mut program: Program, metadata: TransformPluginProgramMetadata) -
         Default::default()
     };
 
+    let module = program.as_module().unwrap();
```

Comment:
Can you use `if let else` to return the `Program::Script` as-is to avoid panic?

Reply:
OK, Solved in 8085c158512812657cc9e008a20d88fc59077e37